_episodes/03-configuration.md

```diff
@@ -12,8 +12,9 @@ objectives:
 
 keypoints:
 - The ``config-user.yml`` tells ESMValTool where to find input data.
-- "``rootpath`` defines the root directory for the input data."
 - "``output_dir`` defines the destination directory."
+- "``rootpath`` defines the root path of the data."
+- "``drs`` defines the directory structure of the data."
 
 ---
 
```

Comment:
I cannot comment on line 26, but I would like to rephrase that. Now, we show a link 'an example file can be found there'. But it's not an example file, it is _the_ default file. So when you type `esmvaltool config get ...`, this is the file that is copied to your home folder. The learners will not need to be able to locate the file in the core package, so why not just change this to: 

"By typing `esmvaltoog config get ...` a copy of the default configuration will be stored in your home directory, where you can then modify it as seen fit."

Reply:
it is revised.